pages/account/deposit.vue

```diff
@@ -237,20 +237,43 @@
                 v-text="t('governance_page_withdraw_like')"
               />
             </h3>
+
+            <!-- Lock Time Warning -->
+            <div
+              v-if="governanceData.isWithdrawLocked.value"
+              class="p-3 rounded-lg bg-amber-50 border border-amber-200"
+            >
+              <div class="flex items-start gap-2">
+                <UIcon
+                  name="i-material-symbols-lock-outline-rounded"
```

Comment:
it seems this is not a valid icon name

https://icones.js.org/collection/material-symbols?s=lock-outline-rounded&variant=Outline+Rounded

Reply:
Maybe use this one

<img width="219" height="106" alt="Image" src="https://github.com/user-attachments/assets/d0977a0d-c69f-4eca-8605-2ffe0bb83ef7" />

```suggestion
                  name="i-material-symbols-lock-clock-outline-rounded"
```